app/pages/privacy-policy.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PrivacyPolicy from './privacy-policy';
+
+describe('PrivacyPolicy', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<PrivacyPolicy />);
+  });
+
+  it('renders without crashing', () => {
+    const privacyPolicyContainer = wrapper.find('div.content-container');
+    assert.equal(privacyPolicyContainer.length, 1);
+  });
+
+  describe('heading', () => {
+    it('renders the page title', () => {
+      const headingElement = wrapper
+        .find('div.content-container').children()
+        .first()
+        .props().component;
+      assert.equal(headingElement, 'h1');
+    });
+
+    it('renders page title content', () => {
```

Comment:
If you're wanting to check the value of a prop using enzyme, you can pass the key of the prop into the method `.prop('key')`: `wrapper.prop('content')`

The selectors for enzyme's find method works a lot like jQuery. Right now it's a bit hard to follow what is being tested here. This instead could be:
```
assert.equal(wrapper.find('Translate').prop('content'), 'privacy.title');
```